velox/exec/benchmarks/PrefixSortBenchmark.cpp

```diff
@@ -0,0 +1,236 @@
+    }
+    std::vector<VectorPtr> children;
+    for (int column = 0; column < rowType->children().size() - 1; ++column) {
+      children.push_back(fuzzerWithNulls.fuzz(rowType->childAt(column)));
+    }
+    children.push_back(
+        fuzzer.fuzz(rowType->childAt(rowType->children().size() - 1)));
+    return std::make_shared<RowVector>(
+        pool, rowType, nullptr, numRows, std::move(children));
+  }
+
+  const std::string testName_;
+  const size_t numRows_;
+  // Rows address stored in RowContainer
+  std::vector<char*> rows_;
+  std::unique_ptr<RowContainer> data_;
+};
+
+class PrefixSortBenchmark {
+ public:
+  PrefixSortBenchmark() : pool_(rootPool()->addLeafChild("PrefixSortTest")) {}
+
+  void runPrefixSort(
+      const std::vector<char*>& rows,
+      RowContainer* rowContainer,
+      const std::vector<CompareFlags>& compareFlags) {
+    PrefixSortConfig prefixSortConfig(1024);
+    PrefixSort prefixSort(
+        pool_.get(), rowContainer, compareFlags, prefixSortConfig);
+    // Copy rows to sortedRows to avoid sort rows already sorted.
+    std::vector<char*> sortedRows = rows;
+    prefixSort.sort(sortedRows);
+  }
+
+  void runStdSort(
+      const std::vector<char*>& rows,
+      RowContainer* rowContainer,
+      const std::vector<CompareFlags>& compareFlags) {
+    std::vector<char*> sortedRows = rows;
+    std::sort(
+        sortedRows.begin(),
+        sortedRows.end(),
+        [&](const char* leftRow, const char* rightRow) {
+          for (vector_size_t index = 0; index < compareFlags.size(); ++index) {
+            if (auto result = rowContainer->compare(
+                    leftRow, rightRow, index, compareFlags[index])) {
+              return result < 0;
+            }
+          }
+          return false;
+        });
+  }
+
+  // Add benchmark manually to avoid writing a lot of BENCHMARK.
+  void benchMark(
+      const std::string& testName,
+      size_t numRows,
+      const RowTypePtr& rowType,
+      int iterateTimes = 1) {
+    // CompareFlags could be same in benchmark.
+    std::vector<CompareFlags> compareFlags;
+    for (int i = 0; i < rowType->children().size(); ++i) {
+      compareFlags.push_back(
+          {true, true, false, CompareFlags::NullHandlingMode::kNullAsValue});
+    }
+    auto testCase =
+        std::make_unique<TestCase>(pool_.get(), testName, numRows, rowType);
+    // Add benchmarks for std-sort and prefix-sort.
+    {
+      folly::addBenchmark(
+          __FILE__,
+          "StdSort_" + testCase->testName(),
+          [rows = testCase->rows(),
+           container = testCase->rowContainer(),
+           sortFlags = compareFlags,
+           iterateTimes = iterateTimes,
+           this]() {
+            for (auto i = 0; i < iterateTimes; ++i) {
+              runStdSort(rows, container, sortFlags);
+            }
+            return 1;
+          });
+      folly::addBenchmark(
+          __FILE__,
+          "PrefixSort_" + testCase->testName(),
```

Comment:
I suggest to add "%" prefix to the name. 

```
"%PrefixSort_" + testCase->testName(),
```

This way, the results will show relative speedup of PrefixSort_xxx vs. baseline StdSort_xxx.